pyexchange/bittrex.py

```diff
@@ -148,23 +67,23 @@ def __init__(self, api_server: str, api_key: str, secret_key: str, timeout: floa
         self.timeout = timeout
 
     def get_markets(self):
-        return self._http_request("GET", "/api/v1.1/public/getmarkets", {})['result']
+        return self._http_request("GET", "/v3/markets", {})
+    
+    def get_precision(self, pair: str):
+        assert(isinstance(pair, str))
+        return self._http_request("GET", f"/v3/markets/{pair}", {})["precision"]
 
     def get_pair(self, pair: str):
         assert(isinstance(pair, str))
-        return next(filter(lambda symbol: symbol['MarketName'] == pair, self.get_markets()))
+        return next(filter(lambda market: market['symbol'] == pair, self.get_markets()))
 
     def get_balances(self):
```

Comment:
Minor point, but can you add the return Types for these functions? Helps with jumping into the codebase later

Reply:
Oh awesome find! Yeah that would be a great addition.